packages/components/src/components/popup/popup-header.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Icon from '../icon';
+import Text from '../text';
+
+const Header = ({ balance, header, header_icon, text_color, title }) => {
+    return header ? (
+        header()
+    ) : (
+        <div>
```

Comment:
Could we add a class name to this? e.g. `dc-popup-header`

Reply:
@aaron-binary why?

Reply:
Semantics 📃